packages/virglrenderer-android/build.sh

```diff
@@ -28,8 +28,21 @@ termux_step_host_build() {
 	if [ $TERMUX_ARCH = arm ]; then
 		CCTERMUX_HOST_PLATFORM=armv7a-linux-androideabi$TERMUX_PKG_API_LEVEL
 	fi
-
+if $TERMUX_ON_DEVICE_BUILD; then
+	local _INSTALL_PREFIX=$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/opt/$TERMUX_PKG_NAME
+	CC=clang
```

Comment:
No need to set `CC` and `CXX`. `$CCTERMUX_HOST_PLATFORM-clang/clang++` also exist when on-device build.